src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/Pages/ResetPassword.razor.cs

```diff
@@ -0,0 +1,136 @@
+﻿using TodoTemplate.Shared.Dtos.Account;
+
+namespace TodoTemplate.App.Pages;
+
+public partial class ResetPassword
+{
+    [Parameter]
+    [SupplyParameterFromQuery]
+    public string? Email { get; set; }
+    [Parameter]
+    [SupplyParameterFromQuery]
+    public string? Token { get; set; }
+    public string? NewPassword { get; set; }
+    public string? ConfirmNewPassword { get; set; }
+
+    public string? NewPasswordErrorMessage { get; set; }
+    public string? ConfirmNewPasswordErrorMessage { get; set; }
+
+    public bool IsSubmitButtonEnabled { get; set; }
+    public bool IsLoading { get; set; }
+
+    [Inject] public HttpClient HttpClient { get; set; } = default!;
+
+    [Inject] public NavigationManager NavigationManager { get; set; } = default!;
+
+    [Inject] public ITodoTemplateAuthenticationService TodoTemplateAuthenticationService { get; set; } = default!;
+
+    public BitMessageBarType ResetPasswordMessageType { get; set; }
+    public string? ResetPasswordMessage { get; set; }
+
+    protected override void OnInitialized()
```

Comment:
shouldn't we use the `OnInitAsync` method instead?

Reply:
We need another issue to make OnInitialized sealed so no one can override it accidentally @msynk @RezaKargar

Reply:
Due to auto decode, I removed OnInitialized() method. @ysmoradi @msynk